schemas/2018-01-01/Microsoft.EventGrid.json

```diff
@@ -60,11 +60,212 @@
         "properties"
       ],
       "description": "Microsoft.EventGrid/topics"
+    },
+    "eventSubscriptions": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.EventGrid/eventSubscriptions"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2018-01-01"
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/EventSubscriptionProperties"
+            }
+          ],
+          "description": "Properties of the event subscriptions"
```

Comment:
Nit: Consider "Properties of the event subscription" instead of "Properties of the event subscription**s**".